packages/dropdown/src/react/__stories__/index.story.js

```diff
@@ -578,3 +580,167 @@ storiesOf('portal', module).add('position', _ => (
+      onChange,
+      onClick,
+      placeholder
+    })
+    return (
+      <Dropdown.Layout
+        disabled={disabled}
+        style={style}
+        className={className}
+        onKeyDown={handleKeyDown}
+        label={<Dropdown.Label label={label} />}
+        menu={
+          <DropdownContext.Provider value={selectedValue}>
+            <Dropdown.Menu
+              inNode={inNode}
+              isOpen={isOpen}
+              menu={menu}
+              menuPosition={menuPosition}
+              onClick={handleMenuChange}
+              onClose={() => {
+                setOpen(false)
+              }}
+              ref={menuRef}
+              width={width}
+            />
+          </DropdownContext.Provider>
+        }
+        subLabel={<Dropdown.SubLabel subLabel={subLabel} />}
+        button={
+          <Dropdown.Button
+            appearance={appearance}
+            disabled={disabled}
+            error={error}
+            isOpen={isOpen}
+            onClick={handleToggleOpen}
+            setMenuPosition={setMenuPosition}
+            size={size}
+            ref={combinedRef}
+            {...rest}
+          >
+            {icon}
+            <div style={{ height: '100%', position: 'relative', flex: 1 }}>
+              <Dropdown.Selected
+                appearance={appearance}
+                label={longestMenuItemState.label}
+                placeholder={placeholder}
+                selectedLabel={selectedLabel}
+                size={size}
+              />
+            </div>
+          </Dropdown.Button>
+        }
+      />
+    )
+  }
+)
+
+const DropdownWithDynamicIcon = () => {
+  const [selected, setSelected] = useState()
+  const values = {
+    channel: {
+      value: 'channel',
+      icon: <Icon.ChannelIcon style={{ marginRight: 8 }} />,
+      label: 'Channel'
+    },
+    analytics: {
+      value: 'analytics',
+      icon: <Icon.AnalyticsIcon style={{ marginRight: 8 }} />,
+      label: 'Analytics'
+    },
+    authorKit: {
+      value: 'authorKit',
+      icon: <Icon.AuthorKitIcon style={{ marginRight: 8 }} />,
+      label: 'Author Kit'
+    },
+    labs: {
+      value: 'labs',
+      icon: <Icon.LabsIcon style={{ marginRight: 8 }} />,
+      label: 'Labs'
+    }
+  }
+  const handleChange = (e, value) => {
+    setSelected(value)
+  }
+  return (
+    <DropdownWithIcon
+      icon={
+        values[selected] ? (
+          values[selected].icon
+        ) : (
+          <div style={{ width: 24, height: 24, marginRight: 8 }} />
+        )
+      }
+      onChange={handleChange}
+      menu={
+        <>
+          {Object.values(values).map(({ value, icon, label }) => (
+            <Dropdown.Item value={value} key={value} icon={icon}>
+              {label}
+            </Dropdown.Item>
+          ))}
+        </>
+      }
+    />
+  )
+}
+
+storiesOf('custom', module)
+  .add('fixed icon', () => (
+    <DropdownWithIcon
+      icon={<Icon.CalendarIcon style={{ marginRight: 8 }} />}
```

Comment:
<img width="269" alt="Screen Shot 2020-09-01 at 4 05 14 PM" src="https://user-images.githubusercontent.com/552044/91910835-ffc5d600-ec6c-11ea-8dfc-3afa455f0f76.png">

Padding left seems really big to me.  Perhaps double check design?

Reply:
It's 16px on a default dropdown 8px on a small. I reduced this as it was larger on on our current dropdowns and didn't match the designs after pairing with Brian to verify.  However it looks like this component's padding has been reduced since I was I on PTO to 12px on default. Will update 👍🏿